### ISSUE TITLE
Where is nexus/testing module mentioned in docs

### ISSUE THREAD

**Issue by ahmedosama5200** [graphql-nexus/nexus#833]
In the tutorial page [here](https://nexusjs.org/docs/getting-started/tutorial/chapter-testing-your-api), it mentioned `nexus/testing` module which I can't find any trace of it.
Was this part of the old nexus framework ? I guess you should update the docs, then

**Comment by tgriesser** [comment]
@jasonkuhrt any ideas?

**Comment by jasonkuhrt** [comment]
Yeah this is from the old Nexus framework! PR welcome @ahmedosama5200, thanks for raising the issue.
